scripts/pre_commit_linter.py

```diff
@@ -1986,12 +1869,15 @@ def _check_bad_patterns(self):
                 python_utils.PRINT('(%s files checked, %s errors found)' % (
                     total_files_checked, total_error_count))
                 python_utils.PRINT(summary_message)
-        self.process_manager['bad_pattern'] = summary_messages
-        _STDOUT_LIST.append(stdout)
+        process_manager['bad_pattern'] = summary_messages
+        global_stdout.append(stdout)
 
     def _check_patterns(self):
         """Run checks relate to bad patterns."""
-        methods = [self._check_bad_patterns, self._check_mandatory_patterns]
+        args = ((_FILES['.js'] + _FILES['.ts']), [], self.process_manager)
```

Comment:
If these args are needed in the method I think they should be passed in explicitly.